src/systems/filecoin_vm/message/message.id

```diff
@@ -26,14 +26,14 @@ type UnsignedMessage struct {
     // Amount of value to transfer from sender's to receiver's balance.
     Value       abi.TokenAmount
 
+    // GasPrice is a Gas-to-FIL cost
+    GasPrice    abi.TokenAmount
+    GasLimit    GasAmount
```

Comment:
No action required but FYI @anorth flagging that the definition of `GasAmount` found here wrapping a BigInt instead of an int64 implies we can't use `abi.GasUnits` here and stay up to spec.  Lotus uses a bigint based type and so this field is serialized as the cbor byte array major type rather than the signed int major type (which is how `abi.GasUnits` serializes).  For go-filecoin to use `abi.GasUnits` type instead of BigInt that type would need to switch to wrapping a big int.

I'm not sure if this is going to require annoying conversions between int64s and bigints but probably isn't too bad since lotus is doing it right now.

Reply:
@anorth just realized that's a gfc internal type so we're all good